hack/assets/assets.go

```diff
@@ -148,12 +144,18 @@ func GetImageList(latest bool, optional bool, kubernetesVersion string, groups [
 }
 
 // GetVclusterImages returns a list of vcluster images
-func GetVclusterImages(cleanTag string) []string {
-	return []string{
-		"ghcr.io/loft-sh/vcluster-oss:" + cleanTag,
-		"ghcr.io/loft-sh/vcluster-pro:" + cleanTag,
-		config.DefaultHostsRewriteImage,
+func GetVclusterImages(latest, optional bool, cleanTag string) []string {
+	images := []string{"ghcr.io/loft-sh/vcluster-oss:" + cleanTag}
+	if !optional {
+		if latest {
```

Comment:
I don't understand use-case for non-optional _and_ non-latest. Aren't those essentially the same thing ?